store/src/db.rs

```diff
@@ -0,0 +1,64 @@
+use crate::config::StoreConfig;
+use crate::migrations;
+use crate::types::BlockNumber;
+use anyhow::anyhow;
+use deadpool_sqlite::{rusqlite::types::ValueRef, Config as SqliteConfig, Pool, Runtime};
+use miden_crypto::hash::rpo::RpoDigest;
+use miden_crypto::utils::{Deserializable, SliceReader};
+use std::fs::create_dir_all;
+use tracing::info;
```

Comment:
minor nit: we usually collapse imports from the same crate into a single import - e.g.,:
```Rust
use crate::{config::StoreConfig, migrations, types::BlockNumber};
```

Reply:
grouped